src/components/toolbar/__tests__/__snapshots__/toolbarFieldDisplayName.test.js.snap

```diff
@@ -6,13 +6,13 @@ exports[`ToolbarFieldDisplayName Component should handle clearing display name t
     [
       {
         "type": "SET_QUERY_CLEAR_INVENTORY_LIST",
-        "viewId": "loremIpsum",
+        "viewId": undefined,
```

Comment:
required: review why this is failing... the component was altered and the test wasn't